src/sqs/github-app-config-payload.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Types for both Cloud and Enterprise server product
+ */
+export type GitHubAppConfigPayload = {
+	/*
+	 * undefined for cloud, real value for GHE
+	 * Otherwise, lost of duplicates values in msg body for cloud (majority traffic)
+	 */
+	gitHubAppConfig?: {
```

Comment:
don't wrap the config into another type.  We also shouldn't create a new TS file just for a single type.  If you want to create a file for multiple types, we just the standard of using the extension `.types.ts` and adding all types related to the file name in here.

It might make sense to add all payload types in here as well.

Reply:
I don't mind having this as its own type. If you search for all places where the github config is used in an SQS payload, this type makes it trivial to find those places, for example. I would err on the side of one type too many than one type too little.

Reply:
Fixed, no wrapping now.